api/http/internal.go

```diff
@@ -55,6 +55,26 @@ func (dev *internalDevice) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// DELETE /tenants/:tenant_id
+// code: 204 - all tenant data removed
+//
+//	500 - internal server error on removal
```

Comment:
interesting formatting ;)

Reply:
I kept it consistent with the rest of the file. do you want me to reformat all?